site/public/css/plagiarism.css

```diff
@@ -281,15 +283,21 @@ main#main.full-screen-mode .plagiarism-result-cont {
         white-space: nowrap;
     }
 
-    /*td .plag-action-btn-cont a:not(:last-child) {*/
-    /*    margin-right: 20px;*/
-    /*}*/
+    /* td .plag-action-btn-cont a:not(:last-child) { */
+
+    /*    margin-right: 20px; */
+
+    /* } */
+
+    /* .plag-data-group { */
+
+    /*    flex-direction: row; */
+
+    /* } */
+
+    /* select[name="user_id_1"], a.toggle { */
 
-    /*.plag-data-group {*/
-    /*    flex-direction: row;*/
-    /*}*/
+    /*    max-width: 150px; */
 
-    /*select[name="user_id_1"], a.toggle {*/
-    /*    max-width: 150px;*/
-    /*}*/
+    /* } */
```

Comment:
Same thing here as above, good to convert these to a singular block comment (or get rid of altogether?)

Reply:
Yep, I'll get rid of this one too.  I was the one who commented it out in the first place and I guess I forgot to remove it.